src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -151,13 +148,7 @@ public ProcessResult updateCase(
                 if (errorMsg.isPresent()) {
                     return new ProcessResult(singletonList(errorMsg.get()), emptyList());
                 } else {
-                    var updatedExceptionRecordData = exceptionRecordFinalizer.finalizeExceptionRecord(
-                        existingCase.getData(),
-                        Long.toString(existingCase.getId()),
-                        CcdCallbackType.ATTACHING_SUPPLEMENTARY_EVIDENCE
-                    );
```

Comment:
Here existing case (instead of exception record) was being finalized.
Luckily, the data from returned model was not used.